src/components/viz/unrootedTree/unrootedTreeLegend.tsx

```diff
@@ -125,19 +133,19 @@ export const ForceGraphLegend = (props: ForceGraphLegendProps) => {
         y1={tickBarStartY + glyphHeight / 2}
         y2={tickBarStartY - glyphHeight / 2}
         stroke="darkgray"
-      />
-      <text
+      /> */}
```

Comment:
It's usually considered not great ("code smell") to leave large swathes of commented-out code for "final" code. It's super common practice to comment-out chunks of code while you work, but normally you want to have that all gone by the time you merge. There's a few reasons for this, but it's not something you _always_ have to follow. If you have a really good reason for keeping it, you might be right, but if it's to hold on to a previous version in case you want to fall back, that's what version control is for.

Reply:
(This also applies to small chunks, even just a single line of commented-out code. It can be hard to tell why it's still there unless you have a lot of context, so the future dev is kind of stuck scratching their head wondering if this is a message from the past, something that was forgotten to be used, or something that was forgotten to be deleted.)